src/lib.rs

```diff
@@ -148,7 +149,9 @@ fn print_stats(stats: &Stats, cpu_time: f64) {
         info!("conflict literals     : {:<12}   ({:4.2} % deleted)", stats.tot_literals, del_percent);
     }
 
-    info!("Memory used           : {:.2} MB", 0.0); // TODO: implement
+    if let Some(mem_used) = mem_used {
+        info!("Memory used           : {:.2} MB", mem_used);
+    }
```

Comment:
The original minisat excludes the memory used line if the platform doesn't support it

 https://github.com/niklasso/minisat/blob/master/minisat/core/Solver.cc#L1000